src/universal/components/AsyncRoute/AsyncRoute.js

```diff
@@ -3,18 +3,29 @@ import {Route} from 'react-router-dom';
 import PropTypes from 'prop-types';
 import Bundle from '../Bundle/Bundle';
 
-const AsyncRoute = ({mod, exact, path, isPrivate, ...extra}) => (
-  <Route
-    exact={exact} path={path}
-    render={({history, location, match}) => (
-      <Bundle extra={extra} history={history} isPrivate={isPrivate} location={location} match={match} mod={mod} />
+const AsyncRoute = ({mod, exact, path, isPrivate, bottom, extraProps}) => {
+  return (
+    <Route
+      exact={exact} path={path}
+      render={({history, location, match}) => (
+        <Bundle
+          bottom={bottom}
+          extraProps={extraProps}
+          history={history}
+          isPrivate={isPrivate}
+          location={location}
+          match={match}
+          mod={mod}
+        />
     )}
-  />
-);
+    />
+  );
+};
 
 AsyncRoute.propTypes = {
+  bottom: PropTypes.bool,
```

Comment:
Yeah, I like `isAbstract`. That's cool.

I'd suggest at [this line](https://github.com/ParabolInc/action/pull/1088/files#diff-e47fd206d5edcd3992b77bd0b4558922R12) it be:

`isAbstractRoute={isAbstract}` so it's clearer in bundle `Bundle` just what that prop is from.